app/templates/collection/view.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="container dwcm-collectionobject-view">
+    <div class="page-header">
+        <h1>
+            {{model.collectionName}}{{if model.catalogNumber (concat ' / ' model.catalogNumber) ''}}
+        </h1>
+        {{t 'fields.labels.cataloger'}} {{model.agent.fullName}}, {{moment-format model.timestampCreated 'LL'}}
+    </div>
+    <div class="row">
+        <div class="col-xs-12">
+            {{#field-group title='form-component-basic-data' collapsable=false as |fg|}}
+                {{#fg.content}}
+                    <ul>
+                        <li>
+                            <span>{{t 'fields.labels.collection'}}:</span> {{model.collectionName}}
+                        </li>
+                        <li>
+                            <span>Collection code:</span> {{model.code}}
```

Comment:
The labels(Collection code, Description, Institution, Discipline) should probably be translated to Swedish as well and rendered using the `t`-helper. Eg.

```
{{t 'fields.labels.collection_code'}}
```